adsws/feedback/config.py

```diff
@@ -12,7 +12,22 @@
 
 FEEDBACK_SLACK_END_POINT = 'https://hooks.slack.com/services/TOKEN/TOKEN'
 FEEDBACK_SLACK_EMOJI = ':interrobang:'
-FEEDBACK_EMAIL = 'adshelp@cfa.harvard.edu'
+DEFAULT_EMAIL = 'adshelp@cfa.harvard.edu'
+# Feedback processing depends on 'origin' attribute supplied in POST data
+FEEDBACK_FORMS_ORIGIN = 'user_submission'
+BBB_FEEDBACK_ORIGIN = 'bbb_feedback'
+# Email template to be applied based on email subject
+FEEDBACK_TEMPLATES = {
+    'Missing References': 'missing_references.txt',
+    'Associated Articles': 'associated_articles.txt',
+    'Updated Record': 'updated_record.txt',
+    'New Record': 'new_record.txt'
+}
```

Comment:
Ideally, we could have all this integrated into one. Regular user feedback can also be processed with its own template and all could share code, except that the feedback forms will not be sent to slack (although we could send to slack just a message saying that a user has submitted data, but not the data itself, have you considered that?).